### ISSUE TITLE
Rendering of intersection between natural=tree_row and barrier=hedge

### ISSUE THREAD

**Issue by Robou** [gravitystorm/openstreetmap-carto#2392]
Hi, 
I think the rendering of the intersections between lines tagged as natural=tree_row and barrier=hedge should be the same as the rendering of 2 tree_rows intersecting each other. 

See the image below where everything is summed up:
![selection_006](https://cloud.githubusercontent.com/assets/6582891/19019929/50e43784-8899-11e6-93f8-c6de31bf1d85.png)
In the example, the hedges end exactly on a node belonging to the tree_row.

The intersection is treated as two transparent layers adding to each other. 
But IMHO, the intersection should simply be the same color, not "the two layers added". Do you see what I mean ?

Or is it a bad mapping idea to finish hedges on tree_rows ?
Thanks

**Comment by math1985** [comment]
No, I think you're right this is a bug. Yet another problem with using transparency.

**Comment by nebulon42** [comment]
Just for clarification: tree_row uses transparency (tree canopy) while hedge does not.

**Comment by kocio-pl** [comment]
Why do we use transparency for tree_row? Is it because we want to indicate that canopy is not solid? Is it the same for individual trees?

**Comment by Tomasz-W** [comment]
related to https://github.com/gravitystorm/openstreetmap-carto/issues/1753

**Comment by jeisenbe** [comment]
This can be fixed by changing tree rows to use a solid green color, without transparency (opacity). This will make them look more similar to linear `natural=wood` or `landuse=forest` features, and that may be a benefit as well.
